docs/guides/databases/redis/install-and-configure-redis-on-ubuntu-20/index.md

```diff
@@ -0,0 +1,269 @@
+2.  Update your Ubuntu packages.
+
+        sudo apt update
+3.  Install Redis using the package installation program.
+
+        sudo apt install redis-server
+{{< note >}}
+If the Redislabs repository is added, apt automatically installs the latest stable version. We do not recommend installing Redis through the Ubuntu default packages, as that might install an older version.
+{{< /note >}}
+
+To install Redis from a downloaded .gz file, follow these steps:
+
+1.  Download the latest stable version from the [*Redis downloads page*](https://redis.io/download) to your computer. Transfer the Redis file to your host via `scp`, `ftp`, or another file transfer method. The following example shows a Redis 6.0.9 executable. Replace the file name with the actual name of the file you are transferring, and substitute your user name and host IP address.
+
+        scp /localpath/redis-6.0.9.tar.gz user@yourhost:~/
+2.  Unzip the file using the tar utility.
+
+        tar xzf redis-6.0.9.tar.gz
+3.  Change directory to the appropriate directory and install Redis. The actual directory name depends upon the exact Redis version.
+
+        cd redis-6.0.9
+        make
+
+## Enabling and Running Redis
+
+Redis-cli is the main command line interface for Redis. Through this utility, you can interactively store and retrieve data, and perform administrative tasks. You can configure Redis by means of the `redis.conf` file, which contains a list of directives. Each directive consists of a variable name, plus a value, such as `supervised systemd`.
+
+1.  Add a directive to allow Redis to start via the system control utility. Edit the redis .conf file at `/etc/redis/redis.conf`, and change the value of the `supervised` directive to `systemd`. This setting is found in the "General" section of the file.
+
+    {{< file "/etc/redis/redis.conf" >}}
+supervised systemd
+{{< /file >}}
+2.  Use `systemctl` to start the Redis service.
+
+        sudo systemctl restart redis.service
+3.  Enter the interactive Redis CLI.
+
+        redis-cli
+4.  Perform a `ping` to test connectivity to the server.
+
+        PING
+    If Redis is running, it returns a `PONG` as a reply.
+
+        PONG
+5.  Use the `SET` command to create a key-value pairing. Redis returns an `OK` response upon a successful set operation.
+
+        SET server:name "fido"
+6. Retrieve the value of the key you previously set.
+
+        GET server:name
+    Redis returns `fido` as the result.
+
+        fido
+
+{{< note >}}
+The redis.conf file contains extensive documentation and many examples. A sample `redis.conf` file can be found on the [*Redis Git Hub site*](https://raw.githubusercontent.com/redis/redis/6.0/redis.conf).
+{{< /note >}}
+
+{{< note >}}
+You can also use `redis-cli` as a function to run any command. Pass in the command and any associated variables as arguments. For example, you can perform a ping with `redis-cli ping`.
+{{< /note >}}
+
+## Configuring a Password for Redis
```

Comment:
This second level heading, and the ACL second level heading further down the guide, might work well as H3s organized under an H2 of "Securing Redis" or something similar.